CL/cl_ext.h

```diff
@@ -1864,23 +1992,17 @@ clGetSemaphoreHandleForTypeKHR(
 
 #endif /* !defined(CL_NO_NON_ICD_DISPATCH_EXTENSION_PROTOTYPES) */
 
-/***************************************************************
-* cl_khr_external_semaphore_dx_fence
```

Comment:
`cl_khr_external_semaphore_dx_fence` has dissapeared after copying over `cl_ext.h` wholesale from the `gen_headers.py` output - However it looks like this was intended https://github.com/KhronosGroup/OpenCL-Docs/pull/1160